src/main/build/debug.h

```diff
@@ -85,7 +85,7 @@ typedef enum {
     DEBUG_CRSF_LINK_STATISTICS_PWR,
     DEBUG_CRSF_LINK_STATISTICS_DOWN,
     DEBUG_BARO,
-    DEBUG_GPS_RESCUE_THROTTLE_PID,
+    DEBUG_ALTITUDE_CONTROL,
```

Comment:
should be moved to the bottom (replacing DEBUG_ALTHOLD)

Reply:
I was thinking that if we don't move it, then the current blackbox viewer should handle the data as if it was the `GPS RESCUE THROTTLE PID` debug, which has exactly the same structure and scaling, and already exists since 4.4 (maybe 4.5 not sure).
The `DEBUG_ALTHOLD` Blackbox debug was added only in 4.6 so can be removed without any problem, and we only need one of them.